src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartNavigation.cshtml

```diff
@@ -0,0 +1,62 @@
+@using Microsoft.AspNetCore.Authorization
+@using OrchardCore.Contents
+@model OrchardCore.Lists.ViewModels.ListPartNavigationViewModel
+@inject OrchardCore.ContentManagement.IContentManager ContentManager
+@inject Microsoft.AspNetCore.Authorization.IAuthorizationService AuthorizationService
+
+<div class="card bg-primary text-right mb-3" style="z-index: 1001">
+    <div class="card-body">
+        <div class="d-flex justify-content-end">
+
+            @if (await AuthorizationService.AuthorizeContentTypeDefinitionsAsync(User, CommonPermissions.ViewContent, Model.ContainedContentTypeDefinitions, ContentManager))
+            {
+                <a class="btn btn-secondary me-1" asp-action="Display" asp-controller="Admin" asp-route-contentItemId="@Model.Container.ContentItemId">@T["List Items"]</a>
```

Comment:
It'll only add the new buttons on the top. It won't override your customization. Then you like the new default, then you can change your customization accordingly. I don't think this would qualify for a real breaking change.